enterprise-suite/gotests/util/lbc/lbc.go

```diff
@@ -4,27 +4,28 @@ import (
 	"strings"
 	"time"
 
+	"github.com/lightbend/console-charts/enterprise-suite/gotests/args"
+
 	"github.com/lightbend/console-charts/enterprise-suite/gotests/util"
 )
 
 const localChartPath = "../../../."
 const lbcPath = "../../../scripts/lbc.py"
 
-func Install(namespace, tillerNamespace string, lbcArgs, helmArgs []string) error {
+func Install(lbcArgs, helmArgs []string) error {
```

Comment:
I think I made this change originally... to move `args` into `testenv`. But now it's back to the way it was :-). It seems redundant to have to specify the namespaces everywhere, as they are fixed based on the installation environment.